sdks/python/apache_beam/yaml/yaml_transform.py

```diff
@@ -115,6 +115,10 @@ def explicitly_empty():
 
 
 def is_explicitly_empty(io):
+  if isinstance(io, str):
```

Comment:
Likewise, inputs should have been normalized before we hit this. is_explicitly_empty should only process normalized inputs, not raw strings.